### ISSUE TITLE
Additional settings to keep data-test attribute at the end of inline attributes

### ISSUE THREAD

**Issue by VladimirCores** [vuejs/eslint-plugin-vue#2324]
**What rule do you want to change?**
I would like to sort Vue component attribute `data-test` always at the end of the list of attributes so it will be always at the same position.

**Does this change cause the rule to produce more or fewer warnings?**
Probably

**How will the change be implemented? (New option, new default behavior, etc.)?**
it would be great to have special order type like "DATA_TEST__ID"

**Please provide some example code that this change will affect:**
![image](https://github.com/vuejs/eslint-plugin-vue/assets/442111/80904f00-e6ce-4092-ac86-2a48992b6e45)

```vue
      <UFormGroup :label="Strings.PASSWORD" name="password" data-test="form-group-username">
        <UInput :disabled="isActionsLocked" type="password" v-model="formState.password" />
      </UFormGroup>
```

**What does the rule currently do for this code?**
It keeps `data-test` attribute always at the end of the list of all inline attributes

**Comment by FloEdelmann** [comment]
Hmm, a group only for `data-test` (and `data-testid`, which is more common) is much more specific than the current groups in the [`vue/attributes-order` rule](https://eslint.vuejs.org/rules/attributes-order.html).

Also, it overlaps with either `ATTR_DYNAMIC` or `ATTR_STATIC`, so the rule would have to include that attributes in those groups if `DATA_TEST_ID` is not specified, and exclude it from those groups if it is specified separately.

But since `data-test` and `data-testid` are commonly used, it might actually be worth it.

An alternative would be to add support for exact attribute matching, which would be more versatile, but also more effort.

**Comment by VladimirCores** [comment]
So, is there any resolution on this one?

**Comment by anne-gropler** [comment]
I am also interested in this.

**Comment by FloEdelmann** [comment]
> is there any resolution on this one?

I'm fine with either solution, as long as it's well tested and backwards-compatible. PRs welcome 🙂
